### PR TITLE
WIP: Add Gschema and Main Window

### PR DESCRIPTION
**Pull request by Abdallah-Moh**

### CODE REVIEW

File: data/meson.build [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+install_data(
+    'photostat.appdata.xml',
+    install_dir: get_option('datadir') / 'metainfo',
+    rename: meson.project_name() + 'appdata.xml'
+)
+
+install_data(
+    'gschema.xml',
+    install_dir: join_paths (get_option('datadir'), 'glib-2.0', 'schemas'),
+    rename: meson.project_name() + 'gschema.xml'
```
Comment (Suzie97) [inline data/meson.build:16]: There should be a `.` before gschema.xml over here.

File: src/Window.vala [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    construct {
+        get_style_context ().add_class ("rounded");
+        var headerbar = new Gtk.HeaderBar ();
+        headerbar.get_style_context ().add_class ("flat");
+        headerbar.show_close_button = true;
+        headerbar.custom_title = new Gtk.Label ("Untitled - Photostat");
+
+        set_titlebar (headerbar);
+    }
+
```
Comment (Suzie97) [inline src/Window.vala:57]: Make sure to check that the window position is saved when the app is quit with `Ctrl + Q`.

File: data/gschema.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Suzie97) [inline data/gschema.xml:1]: 2 space indentation is recommended to use on markup.

**Review by Suzie97 — CHANGES_REQUESTED** [review]

**Review by Suzie97 — APPROVED** [review]

File: src/Window.vala (hunk 2 above)
Comment (Suzie97) [reply src/Window.vala:57]: Can this be solved in this PR or we should look at it in the future?

Comment (Abdallah-Moh) [reply src/Window.vala:57]: I fixed it locally and will push the changes

**Pull request merged**